CHANGELOG.md

```diff
@@ -4,6 +4,48 @@
 
 ### New features
 
+#### Check checkboxes by using the `checkedValues` option
+
+When using the Checkboxes nunjucks macro you can now use `checkedValues` option to indicate which checkboxes should be checked when the page loads.
+
+For example, this would check the checkboxes that have the values 'foo' and 'bar':
+
+```
+checkedValues: ['foo', 'bar']
+```
+
+This is an alternative to setting the `checked` option on each individual checkbox item.
```

Comment:
```suggestion
Use this kind of code so you do not have to set the `checked` parameter on each individual checkbox item.
```
Is the word 'item' adding anything here?

Reply:
I've removed the word item, but have kept the overall sentence structure as I think the suggested wording pushes users too much towards using `values` when both approaches are equally valid. Very happy to talk this one through though if we do want to change it.